<!-- -*- mode: XML -*- -->

<launch>
	
	<node pkg="teresa_wsbs" type="wsbs_controller" name="wsbs_controller" output="screen" respawn="false">
		<param name="xtion_id" value="/depthcamscan_node/scanXtion"/>
		<param name="cmd_vel_id" value="cmd_vel"/>
		<param name="use_leds" value="false"/>
	</node>
   

</launch>
